packages/lib/src/index.ts

```diff
@@ -124,30 +124,20 @@ export {
   useInteraction,
   useModifierKeyPressed,
 } from './interactions/hooks';
-export { MouseButton } from './interactions/models';
-export { getModifierKeyArray } from './interactions/utils';
 export { default as Box } from './interactions/box';
 
+// Constants
 export { COLOR_SCALE_TYPES, AXIS_SCALE_TYPES } from '@h5web/shared';
-
-// Models
 export { INTERPOLATORS } from './vis/heatmap/interpolators';
+
+// Enums
```

Comment:
Separating out enums to avoid falling into the trap of exporting them with `export type`.